charts/azure-k8s-metrics-adapter/values.yaml

```diff
@@ -0,0 +1,75 @@
+# Default values for azure-k8s-metrics-adapter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: jsturtevant/azure-k8-metrics-adapter-amd64
+  tag: latest
```

Comment:
Let's not forget to update this to `0.3.0-alpha.51` at least

Reply:
Addressed